sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -0,0 +1,55 @@
+import multiprocessing
+import pytest
+from unittest.mock import patch, Mock
+from kubeflow.katib import models
+from kubeflow.katib.api.katib_client import KatibClient
+
+@pytest.fixture
+def katib_client():
+    return KatibClient()
+
+@pytest.fixture
+def valid_experiment():
+    return models.V1beta1Experiment(
+        metadata=models.V1ObjectMeta(name="test-experiment"),
+        spec=models.V1beta1ExperimentSpec()
+    )
+
+class TestCreateExperiment:
+    @patch('kubeflow.katib.api.katib_client.client.CustomObjectsApi.create_namespaced_custom_object')
+    def test_create_experiment_success(self, mock_create_namespaced_custom_object, katib_client, valid_experiment):
+        mock_create_namespaced_custom_object.return_value = {"metadata": {"name": "test-experiment"}}
+        experiment = katib_client.create_experiment(valid_experiment)
+        assert experiment == "Experiment default/test-experiment has been created"
```

Comment:
`create_experiment` API doesn't return any info, so you should not check this assert.